### DISCUSSION TITLE
Freezing issue on muiltiple airgradients (typically once a day or so)

### DISCUSSION THREAD

**Discussion by apetryk2** [ajfriesen/ESPHome-AirGradient#11] [General]
Wondering if anyone else is experiencing an issue where their airgradients are freezing using the ESPHOME and airgradient config?

I installed three separate airgradient pro pre-soldered kits in my home.  All three freeze up intermittently.  I've 'solved' the issue by having homeassistant recognize the freeze (by lack of updates) and then power cycle the airgradients by wifi power plug.  A power cycle always fixes the problem.

Not sure where to file a bug report for this issue but wanted to surface the question here to see if there are others experiencing the same thing.

**Comment by ajfriesen** [comment]
My pre-soldered pro kit is running stable since I got it. Maybe its WiFi?
When filing a bug that would most likely be on the esphome repo and not here.
